models/sipnet/R/met2model.SIPNET.R

```diff
@@ -297,22 +324,62 @@ met2model.SIPNET <- function(in.path, in.prefix, outfolder, start_date, end_date
     }
   }
 
-    
-    if (is.null(out)) {
-      out <- tmp
-    } else {
-      out <- rbind(out, tmp)
-    }
-    
+
+    out <- rbind(out, tmp)
+
   }  ## end loop over years
-  
+
   if (!is.null(out)) {
-    
+
+    # Sipnet does not know how to handle missing values in clim files.
+    # -- No, say it louder: Missing values send Sipnet into _infinite loops_.
+    # Let's remove these with warnings, hmm?
+    if (anyNA(out)) {
+      n_all <- nrow(out)
+
+      out <- na.omit(out)
```

Comment:
This isn't valid. If there are any missing met values the run should fail and the user should get a useful error message. If this is happening regularly with reanalysis met then either there's a bug in our workflow or we should be running our existing gapfilling function on reanalysis met, not just site-level observational met (which is the workflow default)

Reply:
Fair point. I was thinking SIPNET doesn't care about the uneven timesteps, but skipped over how that would mean needing to rescale all the cumulative met vars for the obs that now includes the gap time.

I'll convert this to an error